aten/src/ATen/native/quantized/cpu/fused_obs_fake_quant.cpp

```diff
@@ -146,6 +145,14 @@ std::tuple<at::Tensor, at::Tensor> fused_moving_avg_obs_fake_quant_cpu(
     bool symmetric_quant) {
   // Calculate min/max
   auto observe = observer_on.item().toInt();
+  int64_t size = per_row_fake_quant ? self.size(0) : 1;
```

Comment:
nit: might be good to add a comment what this code block is doing, for future readers